zk.js/src/merkleTree/merkleTreeUpdater.ts

```diff
@@ -187,14 +222,15 @@ const checkComputeInstructionsCompleted = async (
   return accountInfo.currentInstructionIndex.toNumber() === 56;
 };
 
+// TODO: investigate
```

Comment:
what do you want to investigate?

Reply:
while debugging with prints I'd seen some stuff that looked confusing mostly about the call flow of the batchupdate, but it works - removed the comment and added to non-prio'd todos